content/en/hub/tasks/advanced/mirror-boot-pool.md

```diff
@@ -26,7 +26,7 @@ consider these caveats:
 <img src="/images/boot-pool-status.png">
 <br><br>
 
-Click <i class="fas fa-ellipsis-v"></i> on a device entry to access actions
+Click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i>&nbsp on a device entry to access actions
```

Comment:
not sure why this was changed? It worked fine.

Reply:
The aria-hidden and title are the accessibility options for screen readers and such.  The trailing space so the : and the o from on had a break between them.   I didn't mean to change fa to fas, that was a typo.